.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Run tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+        vs-version: [latest,50, 49]
```

Comment:
Do we really need to check compatibility with 3 different versions? Latest is fine, but someone would have to update the `50, 49` whenever a new version is released. I don’t want to do that.

Reply:
I'd rather make sure we have some policy that defines which versions each vsutil version supports. Putting it here cements this policy.

I'd like to make sure that a user doesn't have to switch to a new version immediately, as sometimes VapourSynth releases are broken. At the same time I want to have a codified place that authoritatively defines the supported versions.